components/apps.js

```diff
@@ -531,11 +546,12 @@ SteamUser.prototype._getLicenseInfo = async function() {
 	let appids = [];
 
 	for (let pkgid in packages) {
-		((packages[pkgid].packageinfo || {}).appids || []).filter(appid => appids.includes(appid)).forEach(appid => appids.push(appid));
+		((packages[pkgid].packageinfo || {}).appids || []).filter(appid => !appids.includes(appid)).forEach(appid => appids.push(appid));
```

Comment:
Okay, I found something curious. I don't know how it was "working" before, but check this out. 
Before this bugfix, `appids` was always an empty array, because it filter check if appid exists in an empty array, but it should check if appid not exists in this `appids` array.
Turns out it got broken after it was promisified: https://github.com/DoctorMcKay/node-steam-user/blob/50f4b65ec622914465e29c629842e6325c4a0424/components/apps.js#L558